src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    protected String value;
+    protected boolean isPrivate;
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
```

Comment:
Could you have extracted more from the subclasses? Maybe extract the constructor?